networkx/generators/random_graphs.py

```diff
@@ -1318,7 +1403,8 @@ def my_function(b):
 
             return sp.optimize.brentq(my_function, a, 1)
 
-    graph = nx.Graph()
+    graph = nx.empty_graph(create_using=create_using)
+    check_create_using(graph, directed=False, multigraph=False)
```

Comment:
The line 
```
    create_using = check_create_using(create_using, directed=False, multigraph=False)
```
could go at the start of the function. Then here, it would just be
```
    graph = nx.empty_graph(create_using=create_using)
```